specifyweb/frontend/js_src/lib/components/Forms/Save.tsx

```diff
@@ -320,10 +326,20 @@ function useEnabledButtons(tableName: keyof Tables): {
     'disableClone'
   );
   const [disableAdd] = userPreferences.use('form', 'preferences', 'disableAdd');
+
+  const tablesToNotClone = Object.keys(appResourceSubTypes)
+    .filter((key) => appResourceSubTypes[key].name !== undefined)
+    .map((key) => appResourceSubTypes[key]?.name!.toLowerCase());
```

Comment:
(optional) use filterArray instead of using `!`

```ts
  const appResourcesToNotClonse = filterArray(
    Object.keys(appResourceSubTypes)
      .map((key) => appResourceSubTypes[key]?.name?.toLowerCase())
  );
```